api/public/cerbos/request/v1/request.proto

```diff
@@ -272,6 +272,36 @@ message PlaygroundValidateRequest {
     ];
 }
 
+message PlaygroundTestRequest {
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+		json_schema: {
+			description: "Playground test request"
+		}
+    };
+
+    string playground_id = 1;
+
+    repeated PolicyFile policy_files = 2 [
+        (validate.rules).repeated = { min_items: 1, max_items: 30 },
+        (google.api.field_behavior) = REQUIRED,
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "List of policy files."
+            min_items: 1
+            max_items: 30
+        }
+    ];
+
+    repeated PolicyFile test_files = 3 [
```

Comment:
What's the reason for sending test files separately?

The option to run tests from a separate directory is a bit of a legacy hangover. Since we now support the Go convention of defining tests in a file with the `_test` suffix, I am thinking that  we should enforce that in the playground. WDYT?

Reply:
Yup, makes sense.